unime/src-tauri/tests/tests/get_state.rs

```diff
@@ -39,6 +39,34 @@ async fn test_get_state_create_new() {
     .await;
 }
 
+#[tokio::test]
+#[serial_test::serial]
+async fn test_locale_persistency_during_onboarding() {
+    setup_state_file();
+    setup_stronghold();
+
+    // Deserializing the AppStates and Actions from the accompanying json files.
+    let state1 = json_example::<AppState>("tests/fixtures/states/no_profile_nl-NL_redirect_welcome.json");
+    let state2 = json_example::<AppState>("tests/fixtures/states/nl-NL_redirect_me.json");
+    let action = json_example::<Action>("tests/fixtures/actions/create_new.json");
+
+    let container = AppStateContainer(Mutex::new(state1));
+
+    assert_state_update(
+        // Initial state.
+        container,
+        vec![
+            // Get the initial state.
```

Comment:
The information carried by those comments could be expressed by more meaningful variable names.

Reply:
see comment above